sw/device/lib/testing/mock_mmio.h

```diff
@@ -468,23 +368,7 @@ class MmioTest {
  * calls.
  */
 #define EXPECT_MASK8(offset, ...) \
-  EXPECT_MASK32_AT(this->dev(), offset, __VA_ARGS__)
-
-/**
- * Expect an unspecified 16-bit read at the given offset, followed by a write to
- * the same location, with the same value but with some bits changed; the
- * remaining bits must be untouched.
- *
- * The changed bits are specified by a `std::initializer_list<MaskedBitField>`.
- *
- * This function is only available in tests using a fixture that derives
- * `MmioTest`.
- *
- * This expectation is sequenced with all other `EXPECT_READ` and `EXPECT_WRITE`
- * calls.
- */
-#define EXPECT_MASK16(offset, ...) \
-  EXPECT_MASK32_AT(this->dev(), offset, __VA_ARGS__)
+  EXPECT_MASK_INTERAL_(8, this->dev(), offset, __VA_ARGS__)
```

Comment:
I know existing typo but had to point out: INTERAL_

Reply:
I'll fix it in a commit in this PR :)

Reply:
Whoops. =P